File: Two_Sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        retype = []
        for i in range(0,len(nums)):
            for j in range(i+1,len(nums)):
                if nums[i]+nums[j] == target:
                    return [i,j]
                else:
                    continue
        return False


if __name__ =='__main__':
    a = Solution()
    nums = [3,2,4,7]
    target = 9
    print a.twoSum(nums,target)

File: easy/two_sum.cpp
#include <iostream>
#include <vector>
using namespace std;

class Solution {
public:
	vector<int> twoSum(vector<int>& nums, int target) {
		vector <int> temp;
		int length = nums.size();
		for (int i = 0; i < length - 1; i++)
		{
			for (int j = 1; j < length; j++)
			{
				if (((nums[i] + nums[j]) == target) && i != j)
				{
					temp.push_back(i);
					temp.push_back(j);
					return temp;
				}
			}
		}
	}
};